packages/plans-grid/src/plans-grid/index.tsx

```diff
@@ -35,9 +36,14 @@ const PlansGrid: React.FunctionComponent< Props > = ( {
 	onPlanSelect,
 	onPickDomainClick,
 	disabledPlans,
+	singleColumn,
 } ) => {
 	const { __ } = useI18n();
 
+	// Note: singleColumn prop would be always false until "gutenboarding/feature-picker" feature flag is enabled
+	// and Gutenboarding flow is started with ?latest query param
+	singleColumn && console.log( 'display accordion' ); // eslint-disable-line no-console
```

Comment:
Debug code here! 😁

Reply:
Needed to do something with the variable to pass TS check 🙈  It's under feature flag anyway.
Removed in https://github.com/Automattic/wp-calypso/pull/45054/files#diff-55e2d607ea787bc2c8c8b7917ac89d88L45